pkg/components/exec_node_remote.go

```diff
@@ -54,12 +54,31 @@ func NewRemoteExecNodes(
 			Protocol:      corev1.ProtocolTCP,
 		}),
 	)
+
+	var sidecarConfig *ConfigHelper
+	if spec.JobEnvironment != nil && spec.JobEnvironment.CRI != nil {
+		sidecarConfig = NewConfigHelper(
+			&l,
+			proxy,
+			l.GetSidecarConfigMapName(consts.JobsContainerName),
+			commonSpec.ConfigOverrides,
+			map[string]ytconfig.GeneratorDescriptor{
+				consts.ContainerdConfigFileName: {
+					F: func() ([]byte, error) {
+						return cfgen.GetContainerdConfig(&spec)
+					},
+					Fmt: ytconfig.ConfigFormatToml,
+				},
+			})
+	}
```

Comment:
Is it possible to add newBaseExecNode method and move sidecar config there so we don't copypaste it?

Reply:
Let's cleanup all that backlog later.